app/routes/query-internal.js

```diff
@@ -0,0 +1,52 @@
+const { MessageSender } = require('ffc-messaging')
+const { SFD_VIEW } = require('ffc-auth/scopes')
+const { GET, POST } = require('../constants/http-verbs')
+const { createResponseInternal } = require('../data')
+const { getTicket } = require('../data')
+
+module.exports = [{
+  method: GET,
+  path: '/internal/query/{ticketId}',
+  options: { auth: { strategy: 'jwt', scope: [SFD_VIEW] } },
+  handler: async (request, h) => {
+    const data = await getTicket(request.params.ticketId)
+    return h.view('query-internal', { data })
+  }
+},
+{
+  method: POST,
+  path: '/internal/query/{ticketId}',
+  options: { auth: { strategy: 'jwt', scope: [SFD_VIEW] } },
+  handler: async (request, h) => {
+    const sender = new MessageSender({
```

Comment:
This is okay for now but could split the message sender into its own module for reuse. As its a spike its fine for now.

Reply:
Done 😊